app/page.tsx

```diff
@@ -22,10 +23,17 @@ const links = [
 ];
 
 let logoValue = { imgSrc: "tmdblogo.svg", url: "/", altText: "Not Found" };
+let footerValue = {
+  imgSrc: "footerlogo.svg",
+  url: "/footer",
+  imgAlt: "Not Found",
+};
 export default function Home() {
   return (
     <main className="min-h-screen">
       <Header logo={logoValue} links={links} ActionIcon={BiSearchAlt2} />
+      Homepage
+      <Footer logo={footerValue} />
```

Comment:
This is not good Code, please read the Ticket description and comments and implement as per Requirements.